rasa/utils/tensorflow/layers.py

```diff
@@ -806,7 +840,7 @@ def _loss_margin(
 
         return loss
 
-    def _loss_softmax(
+    def _loss_cross_entropy(
```

Comment:
could you please separate this methods into helpers?

Reply:
Done